charts/dotnet-monitor/dockerfile/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+otelcontribcol_linux_amd64 --config /etc/opentelemetry/config.yaml &
+dotnet-monitor collect
```

Comment:
You could remove this line if you use [prometheusexecreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusexecreceiver)

Reply:
how so?